index.html

```diff
@@ -43,23 +44,24 @@ <h1>Books</h1>
         <br>
         <br>
         <br>
+        <!--Button ADD, CLEAR FORM-->
         <button type="button" onclick="addBook();">Add</button>
         <button type="button" onclick="cleanFormInput();">Clear</button>
     </form>
 </div>
-
+      <!--Sorry-->
     <div id="book-save" style="visibility: hidden;"class="save-book save-book-monitor">
```

Comment:
```suggestion
    <div id="book-save-error" style="visibility: hidden;"class="save-error save-book-monitor">
```
change style name accordinly